modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-create-heterogeneous-nodepools-agent-hc_{context}"]
+= Create Heterogeneous NodePools on Agent HostedClusters
+
+This explains how to create heterogeneous nodepools on an agent platform. Please refer to set up the environment using Agent Platform from section 4.6.1, this section only covers the things needed to configure to have heterogeneous nodepools.
+
+.Procedure
+
+. Configure AgentServiceConfig with two heterogeneous architecture OS images:
++
```

Comment:
From the [OCP docs guidelines](https://github.com/openshift/openshift-docs/pull/92359): Do not use more than one command per code block.